docs/api-docs/getting-started/authentication.md

```diff
@@ -1,30 +1,90 @@
 # Authentication
 
+BigCommerce offers a suite of APIs that let you manage store data, sign customers in, make client-side queries for product information, and write apps that integrate third-party services into store operations.
 
+Although each class of APIs configures authentication differently, they are all based on the OAuth2.0 standard.
 
-BigCommerce has five different APIs that let you manage store data, log in customers, make client-side queries for product information, and more. Each requires a different authentication method.
+## Class I: Store Management REST APIs
 
-## REST APIs
+BigCommerce's Store Management REST APIs use OAuth API account access tokens to authenticate requests. Create a store management OAuth API account to generate and manage access tokens, and pass the access token in the header of the request you want to authenticate.
 
-Requests to BigCommerce's V2 and V3 REST APIs require you to pass an access token in the header. For instructions on generating this credential, see [Obtaining Store API Credentials](/api-docs/getting-started/authentication/rest-api-authentication#obtaining-store-api-credentials#obtaining-store-api-credentials).
+For more details, see [Obtaining Store API Credentials](/api-docs/getting-started/authentication/rest-api-authentication#obtaining-store-api-credentials).
 
-## Storefront API
-The Storefront API is unauthenticated, allowing you to make client-side requests for carts, checkouts, and orders using JavaScript. 
+```http title="Store management REST authentication header"
+X-Auth-Token: {{access_token}}
+```
 
+<!-- theme: info -->
+> Legacy API accounts used HTTP basic authentication. They are no longer available to new stores. [Learn more about migrating](/api-docs/getting-started/authentication/rest-api-authentication#migrating-from-legacy-to-oauth). 
 
-## GraphQL Storefront API
-There are two ways to authenticate with the GraphQL API:
-1. Via a [Storefront API token](/api-reference/cart-checkout/storefront-api-token/api-token/createtoken) passed in your request's header.
-2. Passing a token from within a Stencil theme in your request's header.
+## Class II: GraphQL API
 
-For more details, see [GraphQL API Authentication](/api-docs/storefront/graphql/graphql-storefront-api-overview#authentication).
+GraphQL API queries use JWT-style bearer tokens to authenticate. Consult the REST token generation endpoint that corresponds to your GraphQL use case to determine the required OAuth scope before you create a store management OAuth API account to request JWTs for your GraphQL queries. Pass the bearer token in the header of the query you want to authenticate. 

```

Comment:
Doesn't have to be mentioned here, but might be worth noting that developers using Stencil do not have to generate tokens, as a token is pre-generated for them in the stencil context.

Reply:
excellent.

Reply:
done